src/Palette/Palette.js

```diff
@@ -2,30 +2,45 @@ import React from 'react';
 import './Palette.scss';
 import trashBin from '../images/rubbish.png'
 
-const Palette = ({ palette_name, color_1, color_2, color_3, color_4, color_5 }) => {
-  // add removePalette or pass down from ProjectContainer?
-  // if created here import apicall, usedispatch and action
-    const colors =[color_1, color_2, color_3, color_4, color_5];
+const Palette = ({name, id, color_1, color_2, color_3, color_4, color_5}) => {
 
```

Comment:
On line 5, it should actually be palette_name

Reply:
i changed this as well